src/frontend/apps/desk/src/features/mail-domains/access-management/components/ModalRole.tsx

```diff
@@ -7,12 +7,11 @@ import {
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
-import { Box, Text, TextErrors } from '@/components';
+import { Box, TextErrors } from '@/components';
 import { Modal } from '@/components/Modal';
 import { useUpdateMailDomainAccess } from '@/features/mail-domains/access-management';
 
 import { Role } from '../../domains';
-import { useWhoAmI } from '../hooks/useWhoAmI';
```

Comment:
I wonder if we shouldn't totally remove this hook from the codebase, I can see it does not work as expected.

Reply:
I guess ModalDelete as to be refacto as well, it seems to have the same logic. It could be done in another PR though.
https://github.com/numerique-gouv/people/blob/f116f7805f5a22647a8451b3d051cf4052340bc3/src/frontend/apps/desk/src/features/mail-domains/access-management/components/ModalDelete.tsx#L36-L37